.scripts/check-notice-authors.sh

```diff
@@ -25,17 +25,19 @@ exit_code=0
 # Call with an ancestor whereas all commits newer than the ancestor are checked.
 base="$1"
 if [ -z "$base" ]; then
-    commits="$(git rev-list --reverse HEAD)"
+    commits="$(git rev-list --no-merges --reverse HEAD)"
```

Comment:
This change can probably be reverted since the most recent CI script version.

Reply:
Yes since we do not need to check the DCO message for merge commits.  
The case that we have merge commits an a merge request is rare, but could happen.